components/widgets/users/avatar.scss

```diff
@@ -31,3 +32,10 @@
         height: 128px;
     }
 }
+
+.unselectable {
+    -webkit-user-select: none; /* Chrome all / Safari all */
```

Comment:
I think `user-select` should be enough here to cover all browsers that we support.

Reply:
I was seeing issues in the highlighting, so I suggested adding the vendor prefixes not knowing if that was the issue. I'm still seeing highlighting styles on Safari (I realize I already approved, but not sure if the highlighting is in scope for this issue).

First recording is Chrome, second is Safari:
![highlight-post](https://user-images.githubusercontent.com/44858354/81978716-c7788a00-95e0-11ea-9d37-7b41894adb84.gif)

Reply:
Ugh, of course Safari has to be different...